samples/README.DockerHub.md

```diff
@@ -0,0 +1,30 @@
+# Latest Version of Common Tags
+
+The following tags are the latest stable versions of the most commonly used images. The complete set of tags is listed further down.
+
+- [`4.7.1`](https://github.com/Microsoft/dotnet-framework-docker/blob/master/4.7.1-windowsservercore-1709/runtime/Dockerfile)
+- [`3.5`](https://github.com/Microsoft/dotnet-framework-docker/blob/master/3.5-windowsservercore-1709/runtime/Dockerfile)
+
+The [.NET Framework Docker samples](https://github.com/Microsoft/dotnet-framework-docker/blob/master/samples/README.md) show various ways to use .NET Framework and Docker together. See [Building Docker Images for .NET Core Applications](https://docs.microsoft.com/dotnet/framework/docker/) to learn more.
+
+Watch [dotnet/announcements](https://github.com/dotnet/announcements/labels/Docker) for Docker-related .NET announcements.
+
+### Container sample: Run a simple application
+
+Type the following command to run a sample console application:
+
+```console
+docker run --rm microsoft/dotnet-framework-samples
```

Comment:
I just noticed we don't document what `latest` refers to here as well as the .NET Core samples.